package.json

```diff
@@ -60,7 +60,8 @@
     "lint-fix": "eslint --fix --ext .js,.vue parliament viewer notifiers wiseService",
     "lint-fix-viewer": "eslint --fix --ext .js,.vue viewer",
     "lint-fix-parliament": "eslint --fix --ext .js,.vue parliament",
-    "lint-fix-wise": "eslint --fix --ext .js,.vue wiseService"
+    "lint-fix-wise": "eslint --fix --ext .js,.vue wiseService",
+    "viewerdoc": "jsdoc2md --partial viewer/jsdoc/body.hbs viewer/jsdoc/returns.hbs viewer/jsdoc/sig-name.hbs viewer/jsdoc/global-index.hbs --files viewer/apiSessions.js viewer/apiConnections.js viewer/viewer.js > ../arkimeweb/_wiki/api_docs.md"
```

Comment:
Assumes where arkimeweb is installed?

Reply:
Yeah, this is just for testing to make sure this is a viable solution. If you and @awick like it, I'm going to add an argument to pass in the location of the arkimeweb directory

Reply:
can you leave the default to be ../arkimeweb since sane people will use that :)

Reply:
Yeah, I think so!